### ISSUE TITLE
New-OVSnmpV3User created with no securityLevel - Wrong values set to  v3AuthProtocol and v3PrivacyProtocol

### ISSUE THREAD

**Issue by DungKHoang** [HewlettPackard/POSH-HPOneView#538]
### Expected Behavior
Value of v3AuthProtocol and v3PrivacyProtocol should be set to NA instead of NONE when creating snmpV3 user with NO security Level according to the OneView REST API

### Actual Behavior
Value of v3AuthProtocol and v3PrivacyProtocol are set  NONE. 

### Steps to reproduce

```PowerShell
$user3 = New-OVSnmpV3User -username 'snmpv3user3' -SecurityLevel None 
$user3 | fl *
```

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`):  5.20.2597.1857
HPE OneView Appliance Version (`Get-HPOVVersion -ApplianceVer`):  ApplianceVersion: 5.40.00.424900.00
Output from `$PSVersionTable` on your Windows Host: 5.1.14409.1018

```PowerShell
# Output from $PSVersionTable
```

**Comment by DungKHoang** [comment]
in 5.50 library, it still shows NONE instead of NA

**Comment by ChrisLynchHPE** [comment]
A 5.50 library that fixes this issue has not been released yet.

**Comment by ChrisLynchHPE** [comment]
Actually, the value for the properties are supposed to be `none`, not `NA`.   Here is what the UI does to create the same SNMPv3 user:

```json
{
    "type": "Users",
    "userName": "user1",
    "securityLevel": "None",
    "authenticationProtocol": "none",
    "authenticationPassphrase": "",
    "privacyProtocol": "none",
    "privacyPassphrase": ""
}
```
So if anything, the camel case for `NONE` is incorrect by the library.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.0.2689.2416](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.0.2689.2416) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.500/5.0.2689.2416).

I will be releasing updates to the other library versions that also address this.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
